pandas/io/excel.py

```diff
@@ -1579,6 +1579,14 @@ def _convert_to_protection(cls, protection_dict):
 
         return Protection(**protection_dict)
 
+    @classmethod
+    def _to_excel_range(cls, startrow, startcol, endrow, endcol):
+        """Convert (zero based) numeric coordinates to excel range."""
+        from openpyxl.utils.cell import get_column_letter
+        return (f"{get_column_letter(startcol+1)}{startrow+1}"
```

Comment:
So we still have to support Python 3.5 so can't use formatted string literals just yet

Reply:
Too bad, should have realized that